docs/guides/index.md

```diff
@@ -6,7 +6,56 @@ and features.
 ```{toctree}
 :glob:
 :maxdepth: 1
+:hidden:
 
 *
 esp32/README
 ```
+
+## Platform Guides
+
+-   [Android - Building](./android_building.md)
+-   [Apple - Testing with iPhone, iPad, macOS, Apple TV, HomePod, Watch, etc](./darwin.md)
+-   [Espressif (ESP32) - Getting Started Guide](./esp32/README.md)
+-   [Infineon PSoC6 - Software Update](./infineon_psoc6_software_update.md)
+-   [Linux - Simulated Devices](./simulated_device_linux.md)
+-   [mbedOS - Adding a new target](./mbedos_add_new_target.md)
+-   [mbedOS - Commissioning](./mbedos_commissioning.md)
+-   [mbedOS - Platform Overview](./mbedos_platform_overview.md)
+-   [nRF Connect - Android Commissioning](./nrfconnect_android_commissioning.md)
+-   [nRF Connect - CLI Guide](./nrfconnect_examples_cli.md)
+-   [nRF Connect - Configuration](./nrfconnect_examples_configuration.md)
+-   [nRF Connect - Factory Data Configuration](./nrfconnect_factory_data_configuration.md)
+-   [nRF Connect - Platform Overview](./nrfconnect_platform_overview.md)
+-   [nRF Connect - Software Update](./nrfconnect_examples_software_update.md)
+-   [NXP - Android Commissioning](./nxp_k32w_android_commissioning.md)
+-   [NXP - Linux Examples](./nxp_imx8m_linux_examples.md)
+-   [Silicon Labs - Documentation](https://github.com/SiliconLabs/matter#readme)
+-   [Silicon Labs - Building](./silabs_efr32_building.md)
+-   [Silicon Labs - Software Update](./silabs_efr32_software_update.md)
+-   [TI - Platform Overview](./ti_platform_overview.md)
+
+## Development Guides
+
+-   [Access Control](./access-control-guide.md)
+-   [IP Commissioning](./ip_commissioning.md)
+
+## Setup Guides
+
+-   [Open Thread - Hardware suggestions](./openthread_rcp_nrf_dongle.md)
+-   [Open Thread - Setting up a Pi as a border router](./openthread_border_router_pi.md)
+
+## Troubleshooting Guides
+
+-   [Avahi - Troubleshooting](./troubleshooting_avahi.md)
+
+## Tool Guides
+
+-   [chip-tool](./chip_tool_guide.md)
+-   [Python Matter-Repl](./matter-repl.md)
+-   [python-chip-controller - Advanced](./python_chip_controller_advanced_usage.md)
+-   [python-chip-controller - Building](./python_chip_controller_building.md)
+
+## Build Guides
```

Comment:
nit: Taking to account the importance.. can we move Build Guides before Platform Guides and Tool Guides just after Platform Guides?

Reply:
Yes, although I'd like to do this in a separate PR, with some other planned changes.

Reply:
Actually, applied already here.